Adds baka and smug

SlimeEl opened the pull request:
Adds baka and smug endpoints from nekos.life.

Stonedestroyer commented on roleplay/roleplay.py:
```diff
@@ -455,6 +464,48 @@ def __init__(self):
+    @commands.command()
+    @commands.bot_has_permissions(embed_links=True)
+    async def baka(self, ctx, *, user: discord.Member):
+        """Baka someone!"""
```
This could be changed but not sure to what writing would be better there.

lionirdeadman commented:
It's my belief that baka and slap offer the same functionality so I'd like that removed :yum:

SlimeEl committed:
Adds baka and smug

Adds baka and smug endpoints from nekos.life.

SlimeEl committed:
Adds baka and smug

Adds baka and smug endpoints from nekos.life. 
Fix typo.

Fixes a typo. 
Adjust spelling

Adjusts spelling on baka.
Add more smug links.

Adds more smug links to not rely on nekos. 
Adds smug endpoint

Adds the smug endpoint from nekos.life.

SlimeEl committed:
Merge remote-tracking branch 'origin/patch-1' into patch-1
